packages/storybook/stories/va-search.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { VaSearch } from '@department-of-veterans-affairs/web-components/react-bindings';
+import { getWebComponentDocs, propStructure } from './wc-helpers';
+import { generateEventsDescription } from './events';
+
```

Comment:
@andresriveratoro Looks like we are missing: `VaSearch.displayName = 'VaSearch';`